website/docs/r/instance.html.markdown

```diff
@@ -231,6 +233,12 @@ The `enclave_options` block supports the following:
 
 For more information, see the documentation on [Nitro Enclaves](https://docs.aws.amazon.com/enclaves/latest/user/nitro-enclave.html).
 
+### Maintenance Options
+
+The `maintenance_options` block supports the following:
+
+* `auto_recovery` - (Optional) The automatic recovery behavior of the Instance. Can be `"default"` or `"dedicated"`. See [Recover your instance](https://docs.aws.amazon.com/AWSEC2/latest/UserGuide/ec2-instance-recover.html) for more details.
```

Comment:
I believe the two valid values are `default` and `disabled`. `dedicated` is a type of instance tenancy.

Reply:
@ewbankkit pls see ^^, I think documentation probably needs a quick touch up? :)

Reply:
Relates https://github.com/hashicorp/terraform-provider-aws/pull/24462.